A more detailed example for `except:` in `.spectral.yml`

**User story.**
As an API publisher, I want to describe our APIs and lint with spectral. Some of our APIs aren't yet perfect, but I don't want to blanket disable those rules. I spotted that since 5.2.0 we have `except` which is an ideal way to give permission for violations BUT I have no idea how to refer to a path with { in it such as `paths/~1accounts/{api_key}/secrets/get`

**Is your feature request related to a problem?**
I'd like to try using `except` in a common `.spectral.yml` file that Stoplight Studio can understand rather than moving to one-ruleset-per-spec which was our previous plan. I'm struggling to find enough support on using this new feature though :/

**Describe the solution you'd like**
More examples! I am happy to ask lots of questions and the corral the examples into a docs PR if that's a better use of your expertise though :) I have a wide selection of varying API specs that I need to get this working for.

**Additional context**
I am _assuming_ that Stoplight Studio will update spectral and working on `except` will work out fairly soon. If not, it would be good to know and I think we'll just stick with asking people to run a script or push their branch to get the linting working correctly.

Immediately at hand (and I can write a simpler spec but not today):
 - whole real spec https://github.com/Nexmo/api-specification/blob/master/definitions/account.yml
 - `spectral lint definitions/account.yml` - the spectral rule file before I added the except is here: https://github.com/Nexmo/api-specification/blob/master/.spectral.yml and the aim is to only except the stuff that is currently a problem, to avoid any regressions and work through fixing things

Current output:
```
OpenAPI 3.x detected

/home/lorna/projects/nexmo-api-specification/definitions/account.yml
  36:24    error  oas3-valid-oas-content-example  `example` property type should be object
  90:24    error  oas3-valid-oas-content-example  `example` property type should be object
  97:24    error  oas3-valid-oas-content-example  `example` property type should be object
 151:24    error  oas3-valid-oas-content-example  `example` property type should be object
 162:24    error  oas3-valid-oas-content-example  `example` property type should be object
  175:9  warning  operation-description           Operation `description` must be present and non-empty string.
 231:10  warning  operation-description           Operation `description` must be present and non-empty string.
 268:24    error  oas3-valid-oas-content-example  `example` property type should be object
  324:9  warning  operation-description           Operation `description` must be present and non-empty string.
 341:24    error  oas3-valid-oas-content-example  `example` property type should be object
 361:12  warning  operation-description           Operation `description` must be present and non-empty string.

✖ 11 problems (7 errors, 4 warnings, 0 infos, 0 hints)
```

I'd like to silence all of these individually, but I'm not sure how to refer to paths with the placeholders in . Any example would be welcome (I suspect I only need one example to get all of these) and I'll try to put together a smaller sample spec when I have some time.

--- a/docs/guides/6-exceptions.md
+++ b/docs/guides/6-exceptions.md
@@ -20,7 +20,12 @@ except:
 
 <!-- theme: info -->
 > As per the [RFC 6901](https://tools.ietf.org/html/rfc6901#section-3), special characters
-> `~` and `/` have to be escaped to `~0` and `~1` respectively. For instance, the location of the `get` method from a
+> `~` and `/` have to be escaped to `~0` and `~1` respectively where they
+> appear in a path. For instance, the location of the `get` method from a
 > `/todos` path in an openapi document `/root/here.yaml` would be expressed as `/root/here.yaml#/paths/~1todos/get`.
+>
+> An example with more path segments: the location for a `get` method from a
+> `/todos/{todo_id}/labels` path in an openapi document `/root/here.yaml`
+> would be expressed as `/root/here.yaml#/paths/~1todos~1{todo_id}~1labels/get`.
 >
 > *Hint:* Running Spectral cli with the `--format json` parameter is pretty useful to find out the path segments of each result.

